easy_thumbnails/management/commands/thumbnail_cleanup.py

```diff
@@ -1,10 +1,11 @@
 import gc
 import os
 import time
-from datetime import datetime, date, timedelta
+from django.utils.timezone import datetime, timedelta
```

Comment:
I believe that the previous import is correct. `django.utils.timezone` does not export these names (https://github.com/django/django/blob/cb8d7ca0ba0adc4fcefc069eae26f842a15755a1/django/utils/timezone.py#L22-L29). That they happend to be imported there should be considered an implementation detail.